js/app.js

```diff
@@ -44,32 +45,11 @@ async function fetchKontent(api)
 }
 
 
-async function initiateBlogSummaryList()
-{
-    const params = formatString(itemsParams, ["true", pageSize, "post_date[desc]"]);
-    const api = formatString(apiItems, [projectId, params]);
-	const json = await fetchKontent(api);
-
-    if (json.items.length > 0) {
-        var blogSummary = "";
-        for (var i = 0; i < json.pagination.count; i++) {
-            if (json.items[i].system.type === "blog_post") {
-                blogSummary = buildBlogSummary(json.items[i], json.modular_content);
-                document.querySelector("main").appendChild(blogSummary);
-            }
-        }
-        setNextBatchLoad(json.pagination.next_page);
-    } else {
-        var errorMessage = getApiErrorMessage();
-        document.getElementById("loadMoreBlogItems").setAttribute("style", "display:none;");
-        document.querySelector("main").appendChild(errorMessage);
-    }
-}
-
-
-async function loadNextBlogBatch() {
+async function loadBlogItems() {
     var btnLoadMore = document.getElementById("loadMoreBlogItems");
-    const api = btnLoadMore.dataset.next_page;
+    const params = formatString(itemsParams, ["true", pageSize, "post_date[desc]"]) + (tag ? formatString(itemsForTagParams, [tag]) : "");
```

Comment:
Have you considered using template literals? Unless you are wanting to support IE, they are a good idea: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
Interesting... How would I use it in this example?

Reply:
I've sent 2 PRs to this Dev branch with a couple of options to consider. There are many other variants too.

Reply:
How do we merge all three PRs? I have also made the changes you suggested on mine.

Reply:
the idea was that you would choose the bits you wanted. We don't have to merge _all_ of them

Reply:
assuming you have merged what you want now